cluster-autoscaler/core/static_autoscaler_test.go

```diff
@@ -200,7 +201,7 @@ func TestStaticAutoscalerRunOnce(t *testing.T) {
 		lastScaleUpTime:       time.Now(),
 		lastScaleDownFailTime: time.Now(),
 		scaleDown:             sd,
-		podListProcessor:      pods.NewDefaultPodListProcessor(),
+		processors:            ca_processors.TestProcessors(),
```

Comment:
Similar to scale_up_test.go - it would be nice if we didn't have to initialize this in all the places where we don't care about exact values (3 out of 4).
Maybe the tests could use the NewStaticAutoscaler method, this might require more refactoring though.